src/components/cylc/Info.vue

```diff
@@ -54,11 +54,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </v-expansion-panel-title>
         <v-expansion-panel-text>
           <dl>
-            <dt>Title</dt>
-            <dd>{{ taskMetadata.title }}</dd>
-            <v-divider />
-            <dt>Description</dt>
-            <dd><span class="markup">{{ taskMetadata.description }}</span></dd>
+            <dt><Markdown :markdown="'# ' + (taskMetadata.title || '(Title)')"/></dt>
```

Comment:
Should we enable markdown in the title field?

Reply:
I think that it makes sense to make the Title clear, but that could be done with css. I don't see why _not_ though.

Reply:
Here's a contrived why not:

```cylc
[meta]
    title = **My**_C_**Y**_L_ [#](C) :ref:`workflow <cylc.flow>`
```

I think it would be a good idea to keep the workflow and task `[meta]title` to plain text as we might desire other applications within the UI besides the metadata view. Here's a couple of examples:

* https://github.com/cylc/cylc-ui/issues/581
* https://github.com/cylc/cylc-ui/issues/1072